packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -64,6 +64,11 @@ export interface FuzzTestState extends DDSFuzzTestState<SharedTreeFactory> {
 	 * and the fuzz testing model only simulates async transactions.
 	 */
 	transactionViews?: Map<ISharedTree, ITreeViewFork<typeof fuzzSchema.rootFieldSchema>>;
+	/**
+	 * NodeSchema of clients used to typecheck schema during edit generation/reducing.
+	 * TODO: The NodeSchema for each client should be properly updated if "afterSchemaChange" occurs for the client's schema when schemaOps are supported.
+	 */
+	nodeSchemas?: Map<ISharedTree, FuzzNodeSchema>;
```

Comment:
Nit: what does it mean for `nodeSchemas` to not be present? Is that the same as `any`? We should document this.

Reply:
Added comment for what happens when `nodeSchema` is not present